resources/templates/md/posts/2016-04-04-state-management.md

```diff
@@ -0,0 +1,364 @@
+{:title "Clojure state management"
+:description  "How we can manage states in Clojure (how data might change over time), or llamas love candies too :)"
+:layout :post
+:tags  ["Clojure" "ClojureScript" "State management"]
+:toc false
+:author "Dajana Štiberová"
+:image ""}
+
+Long time ago, during the one interview, I was asked to describe the difference between `vars` and `atoms`.
+It was pretty simple question, but at that time it was really hard to describe this difference for me.
```

Comment:
I wouldn't say that is was hard because it brings into question whether we are Clojure hAkkers ;)